lifetimes/utils.py

```diff
@@ -687,3 +693,38 @@ def _concat2(*arrays):
     arrays = [np.array(ar) for ar in arrays]
     arrays = [ar.reshape((-1,1)) if ar.ndim < 2 else ar for ar in arrays]
     return np.concatenate(arrays, axis=-1)
+
+
+def posterior_predictive_deviation(obs_freq: npt.ArrayLike, gen_freq: npt.ArrayLike, obs_rec: npt.ArrayLike, gen_rec: npt.ArrayLike) -> float:
+    """ 
+    In lieu of a traditional posterior predictive check, calculate the standardized wasserstein distance of frequency, weighed by recency. 
```

Comment:
In view?

Reply:
Can you provide more detail? I'll need to merge this PR today, but I can still answer any questions you have and/or create an issue to implement any proposed changes afterward.